src/world_inspector/mod.rs

```diff
@@ -45,6 +45,8 @@ pub struct WorldInspectorParams {
     pub despawnable_entities: bool,
     /// The window the inspector should be displayed on
     pub window: WindowId,
+    /// Filter entities by name.
+    pub name_filter: String
```

Comment:
I think I'd rather have an `Option<String>` where `None` means that no filter UI will be shown. The default can be `Some(String::new())`.

Reply:
I think i have addressed most of your issues in the last 2 commit's. The only problem i see is if we are to use an Option<String> set to an empty String there really is no way to get back to `None` or vice versa, but one could easily add a system that toggle's that with a KeyCode or something if someone wants to be able to do that.